<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarginalizeWithCVClass</class>
 <widget class="QWidget" name="MarginalizeWithCVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarginalizeWithCV</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>OriginalImg</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="pGraphicsViewer" name="graphicsView"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Marginalize</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>API_Marginalize</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>pGraphicsViewer</class>
   <extends>QGraphicsView</extends>
   <header>../GraphicsViewer/pGraphicsViewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MarginalizeWithCV.qrc"/>
 </resources>
 <connections/>
</ui>
